inst/stan/epinowcast.stan

```diff
@@ -39,6 +35,11 @@ data {
   matrix[urds, nrd_effs + 1] rd_fixed; // design matrix for report dates
   int nrd_eff_sds; // number of standard deviations to use for pooling for rds
   matrix[nrd_effs, nrd_eff_sds + 1] rd_random; // Pooling pmf design matrix 
+  // Observations
```

Comment:
Just a general remark: the missingness model is not implemented as optional yet, I would add that in the end.